src/Processors/RouteProcessor.php

```diff
@@ -169,9 +169,11 @@ protected function processRequest(string $method, Stringable $uri, Collection $r
                 ]);
 
                 if ($method === 'GET') {
-                    return $collection->put('url', [
-                        'query' => $rules->map(fn ($value) => array_merge($value, ['disabled' => false])),
-                    ]);
+                    return $collection->mergeRecursive(collect([
```

Comment:
You don't need to create a new collection to use `mergeRecursive()`, a simple array should do.